### ISSUE TITLE
Improving documentation

### ISSUE THREAD

**Issue by kaiiyer** [GACWR/OpenUBA#31]
1. Adding a project wiki
2. Improving the existing documentation and publishing using Github pages for starters
3. For the installation of ELK on platforms like Mac, Linux, Windows we can add an additional section or else we can use a shell script for automating the whole ELK installation process
4. This installation script can be added to docker too

**Comment by Jovonni** [comment]
Good idea. Also, we have to add the react dev server/build commands. In the root Makefile, the commands are:

`make rb # for react build`

`make rd # for the react development server`

After the build, the node server will display the updated react front end.

Regarding ELK, we can write a few scripts to install, and test
